.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "husky"]
+	path = husky
+	url = https://github.com/husky-team/husky.git
```

Comment:
I'll suggest using 471915be (the Husky on Jan 12), and change to 0.1.2 after it's released, because 0.1.2 has some good bug fixes and improvements.